ca/ca.go

```diff
@@ -179,26 +191,61 @@ func (ca *CAImpl) newIntermediateIssuer(root *issuer, intermediateKey crypto.Sig
 	if err != nil {
 		return nil, err
 	}
-	ca.log.Printf("Generated new intermediate issuer with serial %s and SKI %x\n", ic.ID, subjectKeyID)
+	ca.log.Printf("Generated new intermediate issuer %s with serial %s and SKI %x\n", ic.Cert.Subject, ic.ID, subjectKeyID)
```

Comment:
Pebble's startup logging is imo too spammy now.

    Pebble 2020/08/20 20:47:56 Generated new root issuer CN=Pebble Root CA 6ee3a9 with serial 7763e13ec3f24991 and SKI 62675f56d68b4a74e93aaed38bfbfdf375b2480a
    Pebble 2020/08/20 20:47:57 Generated new intermediate issuer CN=Pebble Intermediate CA 6ee3a9 #1 with serial 1368b1b7a776a696 and SKI fd393217f77da95999769563c8db1d24af99a3c2
    Pebble 2020/08/20 20:47:57 Generated new intermediate issuer CN=Pebble Intermediate CA 335fb8 with serial 5413643aad42ad63 and SKI bcc3646f6e7bc060462ea54556c697784395c128
    Pebble 2020/08/20 20:47:57 Generated issuance chain: Pebble Root CA 6ee3a9 -> Pebble Intermediate CA 6ee3a9 #1 -> Pebble Intermediate CA 335fb8
    Pebble 2020/08/20 20:47:57 Generated new root issuer CN=Pebble Root CA 31d5c2 with serial 7d145f4fb93c1812 and SKI 6569baa2cf24f15a95c22a9939e6bce45536c76b
    Pebble 2020/08/20 20:47:57 Generated new intermediate issuer CN=Pebble Intermediate CA 31d5c2 #1 with serial 09f5bc1b9af57921 and SKI 4179a17e4ecfe06a98a659c708966dacdbaf0386
    Pebble 2020/08/20 20:47:57 Generated new intermediate issuer CN=Pebble Intermediate CA 335fb8 with serial 662fb5ce4a0bb6bb and SKI bcc3646f6e7bc060462ea54556c697784395c128
    Pebble 2020/08/20 20:47:57 Generated issuance chain: Pebble Root CA 31d5c2 -> Pebble Intermediate CA 31d5c2 #1 -> Pebble Intermediate CA 335fb8

My personal preference would be to only log the chain and not the individual issuers being generated. But maybe others find it useful, so I welcome a comment on that.

Reply:
I prefer it to be a bit on the too spammy side than having it output too litle. This way, I can always scroll up in the CI logs to see which names are used in all involved intermediate/root certs, if problems arise. And the default values of `PEBBLE_CHAIN_LENGTH` and `PEBBLE_ALTERNATE_ROOTS` are so that there's no spam in the default config.

Reply:
Fully agree that the names should be logged - those would stay as part of the "Generated issuance chain" line. I was more wondering about the individual logging of each serial and SKI. If those are useful, I won't touch it.